### ISSUE TITLE
Cisco AMP for Endpoints Events Input is stuck at "Please wait".

### ISSUE THREAD

**Issue by ThreatHunterDiary** [Cisco-AMP/amp4e_splunk_events_input#3]
![2018-03-15 12_31_05-inputs _ splunk 7 0 0](https://user-images.githubusercontent.com/13478351/37452426-9621cea8-285a-11e8-9afa-bb14741daaee.png)

After installing the Splunk App and performing all the pre-requisite steps such as installing all pip requirements I am still not able to give any Input in the App.

All the tabs, Input, New Input, and Configurations are showing "Please Wait" only and not showing anything else.

Not sure why this issue is coming up.

**Comment by imanassypov** [comment]
Any update on this? Running in similar scenario,

thanks!

**Comment by ThreatHunterDiary** [comment]
It seems like a Version issue. Once I installed the same in different Splunk version (6.6.0 and 7.1.2) it was working fine.

**Comment by imanassypov** [comment]
Thanks, Jet

Ironically I came to same conclusion just minutes before you posted - thank you again for confirming this. It has also worked on 7.1.3

Additional finding I came across while troubleshooting this - not sure if it was only last night, but the api.amp.cisco.com took ridiculously long time to CRUD the event streams, which resulted in timeouts on the splunk app side. Ie when creating a stream, the app would simply sit in ‘creating’ until eventually timing out. 
I used postman in parallel to GET streams while the app was in process of creating them - the lag between event submission and its appearance on console was more than two minutes. Same deal with DELETE api call from postman - more than 1.5 minutes before the event stream disappeared.
